app/src/cli/mod.rs

```diff
@@ -148,9 +148,13 @@ pub struct ProjectCli {
     /// The path or URL of the project to run; if not specified, this will default to the current directory
     pub path: Option<String>,
 
-    /// Build all the assets with full optimization; this will make debugging more difficult
+    /// Build all the assets with debug information; this will make them larger (default for all commands apart from `deploy` and `serve`)
+    #[arg(short, long, conflicts_with = "release")]
+    debug: bool,
+
+    /// Build all the assets with full optimization; this will make debugging more difficult (default for `deploy` and `serve`)
```

Comment:
I'd suggest mentioning both the pros and cons for both of these comments (i.e. debug is "larger and slower but easier to debug" and release is "smaller and faster but harder to debug"), just so that you don't have to look at the other option to see the downside